pyglet/media/player.py

```diff
@@ -1,13 +1,14 @@
 """High-level sound and video player."""
 
-import time
 from collections import deque
+import time
+from typing import Iterable, Optional, Union
 
 import pyglet
 from pyglet.gl import GL_TEXTURE_2D
 from pyglet.media import buffered_logger as bl
 from pyglet.media.drivers import get_audio_driver
-from pyglet.media.codecs.base import Source, SourceGroup
+from pyglet.media.codecs.base import PreciseStreamingSource, Source, SourceGroup
```

Comment:
`SourceGroup`s are tricky. Of course removing the import is easy, but it was considered (and then forgotten) for typing.
They do not inherit from anything and would technically require the typing to read `Union[Source, SourceGroup]` everywhere.
But since i really dislike touching the source inheritance chain for the 2.0.x portion of this, i will simply go ahead and remove the import, acting like they don't exist from a typing perspective.